src/contracts/dependencies/token/transactions/transferFrom.test.ts

```diff
@@ -1,33 +1,32 @@
 import { createQuantity } from '@melonproject/token-math/quantity';
 import { Address } from '@melonproject/token-math/address';
-
 import { initTestEnvironment } from '~/utils/environment/initTestEnvironment';
-import { getGlobalEnvironment } from '~/utils/environment/globalEnvironment';
 import { approve } from '../transactions/approve';
 import { transferFrom } from '../transactions/transferFrom';
 import { deployToken } from '../transactions/deploy';
 import { getToken } from '../calls/getToken';
 
-const shared: any = {};
+describe('transferFrom', () => {
+  const shared: any = {};
 
-beforeAll(async () => {
-  await initTestEnvironment();
-  shared.address = await deployToken();
-  shared.token = await getToken(shared.address);
-});
+  beforeAll(async () => {
+    shared.env = await initTestEnvironment();
+    shared.address = await deployToken(shared.env);
+    shared.token = await getToken(shared.env, shared.address);
+  });
 
-test('transferFrom', async () => {
-  const environment = getGlobalEnvironment();
-  const accounts = await environment.eth.getAccounts();
-  const howMuch = createQuantity(shared.token, '1000000000000000000');
+  it('transferFrom', async () => {
```

Comment:
They are both aliases for the exact same thing. We should go with one (everywhere) for consistency. I chose "it" because that's what most people do in the javascript world for some reason. I am also fine with "test". But lest choose one and keep it.

Reply:
Actually, I slightly prefer `test` because of the above reasons. But since you already changed all of them I'm fine with `it` too and leave it like that. 

It's a pity that we don't use eslint anymore: https://github.com/jest-community/eslint-plugin-jest/blob/HEAD/docs/rules/consistent-test-it.md ;)

Reply:
I switched it to test() to please you ;-)